tests/test_integration_tests.py

```diff
@@ -14,7 +14,10 @@
 
 indicators = list(INDICATORS.values())
 
+out_of_date_database_xfail = pytest.mark.xfail(reason="Out of date database", raises=ValueError, strict=True)
```

Comment:
Tests currently fail because the database is out of date.

This should be a `GrantaMIException`. It's surfacing as a `ValueError` because the deserialization is failing, again as a result of the v2 openapi-common upgrade. When a critical-error occurs, fields are null which shouldn't be according to the spec, and so deserialization fails. See CR-1489 for details.

Reply:
I've updated the database, tests are still failing because of other non-nullable fields which are coming back from the server as null.

Reply:
This is fixed